server/game/cards/09.3-JfS/ShosuroIbuki.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class ShosuroIbuki extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Remove one fate from each other participating character',
+            when: {
+                afterConflict: (event, context) =>
+                    event.conflict.winner === context.player &&
+                    context.source.isParticipating()
+            },
+            gameAction: AbilityDsl.actions.removeFate(context => ({
+                target: context.game.currentConflict.getParticipants(participant => participant.uuid !== context.source.uuid),
+                amount: 1
+            })),
+            effect: 'remove one fate from each other participating character'
```

Comment:
Hmm, this may be better as 'remove 1 fate from Target1, Target2, ...'
i.e.
`effect: 'remove 1 fate from {0}'`
where I think {0} will be `context.targets` (I'm never quite sure if it's {0} or {1} in this scenario)

Reply:
As discussed on discord (some time ago!), we agreed that actually your message is better